computer_vision/cnn_classification.py

```diff
@@ -0,0 +1,50 @@
+# Convolutional Neural Network
+
+# Resources : https://en.wikipedia.org/wiki/Convolutional_neural_network
+# Download dataset from : https://lhncbc.nlm.nih.gov/LHC-publications/pubs/TuberculosisChestXrayImageDataSets.html
+
+# Installing Tensorflow
+# pip install tensorflow
+
+# Installing Keras
+# pip install --upgrade keras
```

Comment:
There's no need to mention what needs to be downloaded. The requirements.txt file is just for that :)

Also, please include the new dependencies in the requirements.txt (alphabetically) so that the tests may pass.

Reply:
okay.  requirement.txt added to project folder

Reply:
Oh, sorry to be not clear. I meant to add it in the project requirements.txt which we already have in TheAlgorithms/Python project root :)

Reply:
No problem. My dependancies are already in root requirement.txt